src/index.js

```diff
@@ -33,16 +38,28 @@ module.exports = options => ({
   },
   // Fix sourcemaps
   result(stylelintResult, filepath) {
+    const interpolationLines = interpolationLinesMap[filepath] || []
     const lineCorrection = sourceMapsCorrections[filepath]
-    const warnings = stylelintResult.warnings.map(warning =>
-      Object.assign({}, warning, {
-        // Replace "brace" with "backtick" in warnings, e.g.
-        // "Unexpected empty line before closing backtick" (instead of "brace")
-        text: warning.text.replace(/brace/, 'backtick'),
-        line: lineCorrection[warning.line]
-      })
-    )
+    const warnings = stylelintResult.warnings
+      .filter(
+        warning =>
+          // Filter indentation warnings generated by interpolations substitution
+          warning.rule !== 'indentation' ||
```

Comment:
I think this would be a lot more readable if you wrote it in the negative way, so the focus is on when we filter it out:
```js
!(warning.rule === 'indentation'  &&
interpolationLines.indexOf(lineCorrection[warning.line]) >= 0)
```